pkg/migration/migration.go

```diff
@@ -613,14 +622,24 @@ func (h *Handler) constructStorageCluster(ds *appsv1.DaemonSet) *corev1.StorageC
 	}
 
 	// Enable CSI
+	csiEnabled := false
+	telemetryEnabled := false
 	for _, c := range ds.Spec.Template.Spec.Containers {
-		if c.Name == "csi-node-driver-registrar" {
-			cluster.Spec.FeatureGates = map[string]string{
-				"CSI": "true",
-			}
-			break
+		switch c.Name {
+		case "csi-node-driver-registrar":
```

Comment:
can we add two const strings for the container names, use here and in portworx code creates the container.